packages/sources/css/src/components/toggle/src/index.css

```diff
@@ -0,0 +1,74 @@
+/* Design tokens */
+@import '@vtmn/css-design-tokens/src/colors';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+
+/* The default switch - the box around the slider */
+.vtmn-toggle_switch {
+  position: relative;
+  display: inline-block;
+  width: rem(56px);
+  height: rem(32px);
+}
+
+/* Hide default HTML checkbox */
+.vtmn-toggle {
+  opacity: 0;
+  width: 0;
+  height: 0;
+}
+
+/* The default slider */
+.vtmn-toggle_slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: var(--vtmn-color_grey-light-1);
+  transition: 0.4s;
+  border-radius: rem(34px);
+}
+
+.vtmn-toggle[disabled] + .vtmn-toggle_slider {
+  cursor: not-allowed;
+  background-color: var(--vtmn-color_grey-light-2);
+}
+
+.vtmn-toggle_slider::before,
+.vtmn-toggle[disabled] + .vtmn-toggle_slider::before {
+  position: absolute;
+  content: '';
+  height: rem(24px);
+  width: rem(24px);
+  left: rem(4px);
+  bottom: rem(4px);
+  background-color: var(--vtmn-color_white);
+  transition: 0.4s;
+  border-radius: 50%;
+}
+
+.vtmn-toggle:not([disabled]) + .vtmn-toggle_slider:hover::before {
+  box-shadow: 0 0 0 rem(12px) rgba(0, 16, 24, 0.05);
+}
+
+.vtmn-toggle:not([disabled]) + .vtmn-toggle_slider:active::before {
+  box-shadow: 0 0 0 rem(12px) rgba(0, 16, 24, 0.15);
+}
+
+.vtmn-toggle:checked:not([disabled]) + .vtmn-toggle_slider:hover::before {
+  box-shadow: 0 0 0 rem(12px) rgba(0, 125, 188, 0.05);
+}
+
+.vtmn-toggle:checked:not([disabled]) + .vtmn-toggle_slider:active::before {
+  box-shadow: 0 0 0 rem(12px) rgba(0, 125, 188, 0.15);
+}
+
+.vtmn-toggle:checked + .vtmn-toggle_slider {
+  background-color: var(--vtmn-color_brand-digital);
+}
+
+.vtmn-toggle:checked + .vtmn-toggle_slider::before {
+  transform: translateX(rem(24px));
+}
```

Comment:
We could have a `:focus` state though it might interfere a lot with the expected design.

We can at least add a `:focus-visible` pseudo-class (same declarations that for `:active` should be cool)

Reply:
Ok thanks :) I put a focus-visible but it may change afterwards according to the designer choice